From a6a0bde0b00eeae6ac8248e1fcd8c0e6cac636e2 Mon Sep 17 00:00:00 2001
From: Michel JAOUEN <michel.jaouen@st.com>
Date: Thu, 23 May 2024 15:39:51 +0200
Subject: [PATCH 0893/1695] core: add quiet option

Since some optional package are checked during build. The option quiet
is added to remove warnings during build

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 07d560a97bd536f93c83aaa075be605a8eb6acbf core: add quiet option]
Change-Id: Ice18b6af1a5c5b1c351b459617ec7af279dc090d
Signed-off-by: Michel JAOUEN <michel.jaouen@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/SCP-firmware/+/383540
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/385196
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/lib/scmi-server/SCP-firmware/CMakeLists.txt | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/core/lib/scmi-server/SCP-firmware/CMakeLists.txt b/core/lib/scmi-server/SCP-firmware/CMakeLists.txt
index 1828964d7..a673eb3c8 100644
--- a/core/lib/scmi-server/SCP-firmware/CMakeLists.txt
+++ b/core/lib/scmi-server/SCP-firmware/CMakeLists.txt
@@ -639,7 +639,7 @@ endforeach()
 # Configure cmake-format.
 #
 
-find_package(CMakeFormat OPTIONAL_COMPONENTS Format Lint)
+find_package(CMakeFormat QUIET OPTIONAL_COMPONENTS Format Lint)
 
 if(TARGET CMakeFormat::Format)
     add_custom_target(
@@ -676,7 +676,7 @@ endif()
 # Configure markdownlint.
 #
 
-find_package(Markdownlint)
+find_package(Markdownlint QUIET)
 
 if(Markdownlint_FOUND)
     add_custom_target(
@@ -721,7 +721,7 @@ endif()
 # Configure yamllint.
 #
 
-find_package(Yamllint)
+find_package(Yamllint QUIET)
 
 if(Yamllint_FOUND)
     add_custom_target(
@@ -738,7 +738,7 @@ endif()
 # Configure Clang-Format.
 #
 
-find_package(Clang OPTIONAL_COMPONENTS Format FormatGit)
+find_package(Clang QUIET OPTIONAL_COMPONENTS Format FormatGit)
 
 if(Clang_FormatGit_FOUND)
     add_custom_target(
-- 
2.47.2

